tensorflow/lite/tools/versioning/runtime_version.cc

```diff
@@ -73,6 +73,7 @@ std::string FindMinimumRuntimeVersionForOp(tflite::BuiltinOperator op_code,
            {{BuiltinOperator_CONV_2D, 5}, "2.4.0"},
            {{BuiltinOperator_CONV_2D, 6}, "2.9.0"},
            {{BuiltinOperator_CONV_2D, 7}, "2.11.0"},
+           {{BuiltinOperator_CONV_2D, 8}, "2.12.0"},
```

Comment:
@SaoirseARM 
Could you update it to "2.13.0" like `BuiltinOperator_TRANSPOSE_CONV`?